src/services/units.service.ts

```diff
@@ -0,0 +1,23 @@
+const units = [{ $id: 1, name: 'Stk', longForm: "Stück" }, { $id: 2, name: 'g', longForm: "Gramm" }, { $id: 3, name: 'kg', longForm: "Kilogramm" }, { $id: 4, name: 'ml', longForm: "Milliliter" }, { $id: 5, name: 'l', longForm: "Liter" }];
+
+export const UnitService = {
+    getUnitArray() {
```

Comment:
Suggestion: Use `getUnits()` (plural form and remove the Array).
And add a return type f.e. `: IUnit[]`
```suggestion
    getUnits() {
```